packages/dialtone-vue3/components/list_item/list_item.vue

```diff
@@ -196,6 +204,10 @@ export default {
           this.onMouseLeave(event);
           this.$emit('mouseleave', event);
         },
+
+        click: event => {
+          this.$emit('click', event);
+        },
```

Comment:
This was emitting a warning

Reply:
What was the warning? There might be a problem with doing this because a list item can be non-interactive in which case we don't want to emit a click event.

Reply:
The error was that it was emitting a click event but click wasn't declared on 'emits'

Reply:
https://github.com/dialpad/dialtone/assets/87546543/a1b4d88c-a8dd-490b-b55e-478e748e3323

Only firing the event while pressing `enter` or `space` on a list-item